### PR TITLE
Added new required functions

### PR DESCRIPTION
**Pull request by Banner-Keith**
Return the value instead of the value or undefined.

getInputRequired
getPathInputRequired
getEndpointUrlRequired
getEndpointDataParameterRequired
getEndpointAuthorizationSchemeRequired
getEndpointAuthorizationParameterRequired

### CODE REVIEW

**Comment by Banner-Keith** [comment]
Fixes #824 

I also saw that #826 was opened. I can rebase after that PR is completed.

I have added unit tests for everything I added and all tests are passing.

**Comment by Banner-Keith** [comment]
`getInputRequired` and `getPathInputRequired` are the important ones but I did the same addition on some other similar methods. If you only want some of those I can remove some of the additions.

I can also adjust naming or anything else if there's a suggestion.

File: node/task.ts [hunk 1]
```diff
@@ -228,6 +228,24 @@ export function getInput(name: string, required?: boolean): string | undefined {
...
 
+/**
+ * Gets the value of an input.
+ * If the value is not set, it will throw.
+ *
+ * @param     name     name of the input to get
+ * @returns   string
+ */
+export function getInputRequired(name: string): string {
+    var inval = im._vault.retrieveSecret('INPUT_' + im._getVariableKey(name));
```
Comment (anatolybolshakov) [inline node/task.ts:239]: Please use the call of existing methods instead (like getInput(name, required) - to avoid duplication

**Comment by anatolybolshakov** [comment]
Thanks for contribution! Could you please take a look at comments?

File: node/task.ts (hunk 1 above)
Comment (Banner-Keith) [reply node/task.ts:239]: Do you mean that I should have `getInputRequired` call `getInput` so that the internals of that method are not duplicated?

Comment (anatolybolshakov) [reply node/task.ts:239]: @Banner-Keith yep correct 👍

**Review by SvetlanaMaliugina — APPROVED** [review]
Tested 'getInputRequired' and 'getPathInputRequired' methods manually. Looks fine for me.

**Comment by anatolybolshakov** [comment]
@mmrazik could you please take a look also?

**Pull request merged**